packages/flutter/test/widgets/image_rtl_test.dart

```diff
@@ -301,7 +298,9 @@ void main() {
     expect(find.byType(Container), isNot(paints..drawImageRect()..drawImageRect()));
   });
 
-  testWidgets('Image RTL with alignment topEnd and match', (WidgetTester tester) async {
+  testWidgets('Image RTL with alignment topEnd and match',
+  experimentalLeakTesting: LeakTesting.settings.withIgnoredAll(), // The test leaks by design, because of hacky way dealing with images.
```

Comment:
What is hacky about this?

Reply:
It uses TestImageProvider that overrides `loadImage` that just creates new ImageInfo without taking care about disposal.

Reply:
Normally leak tracker ignores leaks created by test helpers. Images is exception because some part of image is created natively and it is not recognizable as created in test helper.

Reply:
Can you review documentation update to reflect this: https://github.com/dart-lang/leak_tracker/pull/246

Reply:
> add complexity to this class to take care about disposal.

How complex would it be to handle it properly?

Reply:
I fixed some and explain hackiness of others. I believe no one should spend time figuring out how to dispose images  for this level of hackiness.